packages/commons/src/context/context.ts

```diff
@@ -26,12 +26,19 @@ export function fromAppContext(ctx: AppContext): WithLogger<AppContext> {
 }
 
 export const contextMiddleware =
-  (serviceName: string): ZodiosRouterContextRequestHandler<ExpressContext> =>
+  (
+    serviceName: string,
+    overrideCorrelationId: boolean = false
+  ): ZodiosRouterContextRequestHandler<ExpressContext> =>
   (req, _res, next): void => {
+    const correlationId = !overrideCorrelationId
```

Comment:
isn't it the opposite?
```suggestion
    const correlationId = overrideCorrelationId
```

Reply:
🙏🏻  mistake in my brain